main/src/library/Graph.flix

```diff
@@ -15,61 +15,55 @@
  */
 namespace Graph {
 
+    def nodes(): #{ Edge(t, t), Node(t) | r } with Boxable[t] = #{
```

Comment:
- Add documentation comments.
- Private functions belong at the bottom of the file or immediately after the public function where they are used :)

Reply:
It makes most sense to have the private functions at the bottom of the file, when they are used by multiple public functions, right?

Reply:
Yep